### ISSUE TITLE
Tile function is not working.

### ISSUE THREAD

**Issue by tanokana** [playcanvas/engine#4212]
Engine Version 1.52.6.
I used "Tile" because I wanted to express the same pattern repeating.
Placed it under 2D Screen with Element→Type "Image".
It looked fine.

However, when Engine became Version 1.53.2
I started getting the following error.

```
Failed to compile fragment shader:

ERROR: 0:210: 'texture' : no matching overloaded function found
ERROR: 0:210: 'a' : field selection requires structure, vector, or interface block on left hand side

205: #ifdef MAPFLOAT
206: dAlpha *= material_opacity;
207: #endif
208:
209: #ifdef MAPTEXTURE
210: dAlpha *= texture2D(texture_opacityMap, nineSlicedUv, -1000.0, textureBias).a;
211: #endif
212:
213: #ifdef MAPVERTEX
214: dAlpha *= clamp(vVertexColor.VC, 0.0, 1.0);
215: #endif

[object Object]

Error in material "defaultScreenSpaceImage9TiledMaterial" with flags 65797
```

I have created a project to verify the issue.
https://playcanvas.com/project/919074/overview/tiletexture

I have tried Image, Sprite, Texture, and I tried Repeat and Clamp ......

**Comment by mvaligursky** [comment]
Thanks, we'll be fixing this today.

**Comment by willeastcott** [comment]
Fixed by #4214

**Comment by tanokana** [comment]
![image](https://user-images.githubusercontent.com/104486697/165684072-2d245bc2-f452-43fc-8222-36dbeb5ff4e3.png)
Thank you!
